plugin/al/lib/AL_USDMaya/CMakeLists.txt

```diff
@@ -194,7 +194,7 @@ target_link_libraries(${LIBRARY_NAME}
     AL_EventSystem
     AL_USDMayaUtils
     AL_MayaUtils
-    AL_USDUtils
+    USDUtils
```

Comment:
Following on from comment above about API breaking changes being noted in the PR description - a change to the library name merits the same treatment